src/states/GlobalContext.js

```diff
@@ -0,0 +1,21 @@
+import React, { createContext, useReducer } from 'react';
+import styled, { ThemeProvider } from 'styled-components';
+import { initialState, globalReducer } from '../utils/reducers/globalReducer';
+
+const Background = styled.div`
+  background-color: ${({ theme }) => theme.background};
+`;
+
+export const GlobalContext = createContext(initialState);
+
+export function ContextWrapper({ children }) {
+  const [state, dispatch] = useReducer(globalReducer, initialState);
+
+  return (
+    <GlobalContext.Provider value={[state, dispatch]}>
```

Comment:
It is easier to read your context as `const { state } = useContext(GlobalContext)`, also it will help you keep your state organized as you can extract exactly what you need from it. It doesn't make much of a difference right now, as you are just passing down what you get from the reducer. But if you happen to have more state data, the array approach will be a lot harder to access and maintain.